experimental/schoology/main.py

```diff
@@ -12,7 +12,7 @@
 import os
 
 from dotenv import load_dotenv
-import pandas as pd
+import pandas as pd  # type: ignore

```

Comment:
should be able to remove these now

Reply:
I left this one in the experimental/schoology file since this project doesn't have a mypy ini

Reply:
sorry, thought it was the google main.py 🙃